packages/metastream-app/src/locale/ko-KR.ts

```diff
@@ -6,15 +6,21 @@ export default {
   allow: '허용',
   allowTracking: '앱 사용 정보 보내기',
   allowTrackingDesc: '익명 사용 정보는 개발자가 앱이 어떻게 사용되는지 알기 위해 전송됩니다.',
+  appearance: 'Appearance',
```

Comment:
Should this also be translated?

Reply:
I'm sorry, I forgot to translate it.
I'll fix it and pull request again.

Reply:
You should be able to make changes without creating a new pull request. Here's a video tutorial i found which goes through it https://www.youtube.com/watch?v=e41HPOHX9aE

Reply:
I fixed it. Thanks!